### ISSUE TITLE
an error while importing

### ISSUE THREAD

**Issue by liuranranranran** [mcfletch/pyopengl#132]
The following error occurred when I imported PyOpenGL
System: Ubuntu20.04
Python: 3.8.19
PyOpenGL and PyOpenGL_accelerate: 3.1.7
How could I fix it
![17351081238367](https://github.com/user-attachments/assets/7559f466-c803-428b-ba4e-6d8b2a7452c4)

**Comment by mcfletch** [comment]
Somehow your machine wasn't able to find the OpenGL library, which is a fairly rare occurrence on real Ubuntu boxes. Do you *really* mean 20.04? Not 22.04 or 24.04? Is it possible that you're running inside a container that has no OpenGL implementation?

**Comment by liuranranranran** [comment]
> Somehow your machine wasn't able to find the OpenGL library, which is a fairly rare occurrence on real Ubuntu boxes. Do you _really_ mean 20.04? Not 22.04 or 24.04? Is it possible that you're running inside a container that has no OpenGL implementation?

yes,i'm running inside a container.But, i've solved it by unintsalling pyopengl and the packages that may be associated with pyopengl and reinstalling them with conda
